etc/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+install(DIRECTORY workloads/
+        DESTINATION etc/genny/workloads
+        COMPONENT Workloads
+        FILES_MATCHING
+            PATTERN "*.yml"
+            PATTERN "*.yaml"
+)
```

Comment:
so this makes there two directories where workloads live....we have a ticket (somewhere...) to move the workloads directory to a more sensible place - let's kill this change for now and keep things together.

Reply:
I think the install change is still worth it to me because it allows separating the workloads by suite. The extra `etc` directory not so much.